apps/microsoft-teams/frontend/src/components/config/NotificationEditModeFooter/NotificationEditModeFooter.tsx

```diff
@@ -7,16 +7,21 @@ interface Props {
   handleCancel: () => void;
   handleSave: () => void;
   isSaveDisabled: boolean;
+  isTestLoading: boolean;
 }
 
 const NotificationEditModeFooter = (props: Props) => {
-  const { handleTest, handleCancel, handleSave, isSaveDisabled } = props;
+  const { handleTest, handleCancel, handleSave, isSaveDisabled, isTestLoading } = props;
 
   return (
     <Box className={styles.footer}>
       <Flex justifyContent="flex-end" margin="spacingS">
         <ButtonGroup variant="spaced" spacing="spacingS">
-          <Button variant="transparent" onClick={handleTest}>
+          <Button
+            variant="transparent"
+            onClick={handleTest}
+            isLoading={isTestLoading}
+            isDisabled={isTestLoading}>
```

Comment:
Would it make more sense  if `isTestLoading` to use `Sending` verbage instead of `Send test message`? No biggie open to either.

Reply:
I like changing this to `isTestSending`, thanks for the suggestion! 🙌